addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -27,6 +28,16 @@
     &:not(.o-composer-is-compact) {
         // Don't allow the input to take the whole height when it's not compact
         // (like in chatter for example) but allow it to take some more place
-        max-height: 150px;
+        max-height: 400px;
     }
 }
+
+.o_ComposerTextInput_textarea_dummy {
+    position: absolute;
+    opacity: 0;
+    border-top-width: 0;
+    border-bottom-width: 0;
```

Comment:
Changing the border size does not look like a good idea when computing height. Would be best if it was removed from here, and the compute method took into account `clientHeight` or `offsetHeight`, as necessary.

Reply:
Copied from `web.dom`. But work's the same if removed.

Reply:
Make sure the code is robust enough to also work if we change the border size and/or padding of the textarea to either 0 or non-0 values.